### DISCUSSION TITLE
Gitea / Forgejo Actions manager

### DISCUSSION THREAD

**Discussion by morremeyer** [renovatebot/renovate#23607] [Suggest an Idea]
### Tell us more.

Forgejo now supports actions, documented at https://forgejo.org/docs/latest/user/actions/ and https://forgejo.org/docs/latest/admin/actions/.

The documentation is very explicity about the syntax being similar, but not exactly the same as GitHub actions.

For now, the gitub-actions manager should work fine with forgejo actions files when expicitly configured, but that might change.

Do we want to start having a foregejo-actions manager, too, from early on? I'd be up to open a PR and implement the initial version for it, but want to gather some input first

**Comment by viceice** [comment]
yes, but it should also support Gitea.

and we probably first need a gitea-releases / gitea-tags datasource

**Reply by morremeyer** [reply]
So as long as forgejo tracks gitea, let's use a gitea-releases/gitea-tags datasource and a gitea-actions manager.

If forgejo ever hard-forks, we can then split it into forejo-* and gitea-* once needed.

Did I get that correct?
If yes, I might start to look into it soon-ish since I'd like to have that support for my own forgejo instance.

**Reply by viceice** [reply]
yes. 👍🤗
